components/dashboard/src/App.tsx

```diff
@@ -4,188 +4,191 @@
-        }
-    ];
-
-    if (user && user?.rolesOrPermissions?.includes('admin')) {
-        list.push({
-            title: 'Admin',
-            link: '/admin',
-            alternatives: adminMenu.flatMap(e => e.link)
-        });
-    }
-
-    list.push({
-        title: 'Docs',
-        link: 'https://www.gitpod.io/docs/',
-        alternatives: []
-    });
-    list.push({
-        title: 'Community',
-        link: 'https://community.gitpod.io/',
-        alternatives: []
-    });
-
-    return <Menu list={list}/>;
-}
-
-export default App;
+ import React, { Suspense, useContext, useEffect, useState } from 'react';
+ import Menu from './components/Menu';
+ import { BrowserRouter } from "react-router-dom";
+ import { Redirect, Route, Switch } from "react-router";
+ 
+ import { Login } from './Login';
+ import { UserContext } from './user-context';
+ import { getGitpodService } from './service/service';
+ import { shouldSeeWhatsNew, WhatsNew } from './WhatsNew';
+ import settingsMenu from './settings/settings-menu';
+ import { User } from '@gitpod/gitpod-protocol';
+ import { adminMenu } from './admin/admin-menu';
+ 
+ const Workspaces = React.lazy(() => import(/* webpackPrefetch: true */ './workspaces/Workspaces'));
+ const Account = React.lazy(() => import(/* webpackPrefetch: true */ './settings/Account'));
+ const Notifications = React.lazy(() => import(/* webpackPrefetch: true */ './settings/Notifications'));
+ const Plans = React.lazy(() => import(/* webpackPrefetch: true */ './settings/Plans'));
+ const Teams = React.lazy(() => import(/* webpackPrefetch: true */ './settings/Teams'));
+ const EnvironmentVariables = React.lazy(() => import(/* webpackPrefetch: true */ './settings/EnvironmentVariables'));
+ const Integrations = React.lazy(() => import(/* webpackPrefetch: true */ './settings/Integrations'));
+ const Preferences = React.lazy(() => import(/* webpackPrefetch: true */ './settings/Preferences'));
+ const StartWorkspace = React.lazy(() => import(/* webpackPrefetch: true */ './start/StartWorkspace'));
+ const CreateWorkspace = React.lazy(() => import(/* webpackPrefetch: true */ './start/CreateWorkspace'));
+ const InstallGitHubApp = React.lazy(() => import(/* webpackPrefetch: true */ './prebuilds/InstallGitHubApp'));
+ const FromReferrer = React.lazy(() => import(/* webpackPrefetch: true */ './FromReferrer'));
+ const UserSearch = React.lazy(() => import(/* webpackPrefetch: true */ './admin/UserSearch'));
+ const WorkspacesSearch = React.lazy(() => import(/* webpackPrefetch: true */ './admin/WorkspacesSearch'));
+ 
+ function Loading() {
+     return <>
+     </>;
+ }
+ 
+ function App() {
+     const { user, setUser } = useContext(UserContext);
+ 
+     const [loading, setLoading] = useState<boolean>(true);
+     const [isWhatsNewShown, setWhatsNewShown] = useState(false);
+ 
+     useEffect(() => {
+         (async () => {
+             try {
+                 const usr = await getGitpodService().server.getLoggedInUser()
```

Comment:
<details>
<summary>:warning: <strong>This code contains a statement without a semicolon.</strong> (click for details)</summary>

A possible improvement would be to keep the code consistency and insert the semicolon at the end of the line.  
  
This change would ensure the code follows the current code standard, making it easier for new developers to start without the need to learn a different convention and, thus, improving maintainability and keeping the code consistency.

**[More details](https://confluence.devfactory.com/x/lAsaF)**

## The improvement suggestion in this case
Consider adding the semicolon at the end of the line.  
  
```suggestion  
                 const usr = await getGitpodService().server.getLoggedInUser();
```
</details>

[//]: # (<a id="623b5be1-8d15-408b-b60f-a58e03b87463" data-finding-id="337251220" href="none" codereview-uuid-reserved-tag data-business-id="507be860-41ae-46a0-83e9-7f8e359e3039" data-analyzer="js" />)